buildpacks/go/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for `arm64` and multi-arch images. ([#261](https://github.com/heroku/buildpacks-go/pull/261))
```

Comment:
You'll need to release actions first, before releasing this, xref:
https://github.com/heroku/buildpacks-procfile/pull/225#issuecomment-2091769230

Reply:
Got it, will do.